src/test/resources/pages/InstructorCourseStudentEditPage.html

```diff
@@ -112,7 +112,7 @@
         <input name="courseid" type="hidden" value="CCSDEditUiT.CS2104">
         <div class="form-group">
           <label class="col-sm-1 control-label">
-            Student Name:
+            Name:
```

Comment:
As before - let's not change this within this PR, not within the scope of the PR